src/main.rs

```diff
@@ -96,6 +105,41 @@ async fn main() -> std::io::Result<()> {
         .new_game(GameOptions {}, Some("test".to_string()))
         .unwrap();
 
+    let reaper_manager_ref = game_manager_wrapper.clone();
+
+    // Game Reaper thread
+    thread::spawn(move || loop {
+        thread::sleep(Duration::from_secs(REAP_INTERVAL));
+        let test_handle = GameHandle("test".to_string());
+        let num_games = reaper_manager_ref
+            .game_manager
+            .read()
+            .unwrap()
+            .games
+            .len()
+            .clone();
+
+        let mut game_manager_mut = reaper_manager_ref.game_manager.write().unwrap();
+        let games = game_manager_mut.games.clone();
```

Comment:
You don't really need to clone this, you could just iterate through it on line 126 and clone the `game_wrapper` when you insert it into unreaped games.